src/routes/userRoute.js

```diff
@@ -0,0 +1,13 @@
+import { Router } from 'express'
+const router = Router()
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
const router = Router();
```